modules/rest/src/main/java/org/apache/ignite/internal/rest/deployment/DeploymentManagementController.java

```diff
@@ -123,4 +100,11 @@ public static UnitStatus fromUnitStatus(org.apache.ignite.internal.deployunit.Un
         }
         return new UnitStatus(status.id(), versionToDeploymentStatus);
     }
+
+    private static Version parseVersion(String unitVersion) {
+        if (unitVersion == null || unitVersion.isBlank()) {
```

Comment:
Why is this logic located in `DeployManagementCongroller`? I think this is `Version.parseVersion` responsibility.